draft-ietf-ppm-dap-taskprov.md

```diff
@@ -595,8 +603,69 @@ out once they have opted in.
 
 # IANA Considerations
 
-> NOTE(cjpatton) Eventually we'll have IANA considerations (at the very least
-> we'll need to allocate a codepoint) but we can leave this blank for now.
+This document requests a codepoint for the `taskbind` extension and for
+creation of a new registry for DP mechanisms.
+
+(RFC EDITOR: Replace "XXXX" with the RFC number assigned to this document.)
+
+## Report Extension
+
+The following entry will be (RFC EDITOR: change "will be" to "has been") added
+to the "Report Extension Identifiers" registry of the "Distributed Aggregation
+Protocol (DAP)" page created by {{!VDAF}}:
+
+Value:
+: `0xff00`
+
+Name:
+: `taskbind`
+
+Reference:
+: RFC XXXX
+
+## Registry for DP Mechanisms
+
+A new registry will be (RFC EDITOR: change "will be" to "has been") created for
+the "Distributed Aggregation Protocol (DAP)" page called "Differential Privacy
+(DP) Mechanisms". This registry contains the following columns:
+
+Value:
+: The one-byte identifier for the DP mechanism
+
+Name:
+: The name of the DP mechanism
+
+Reference:
+: Where the DP mechanism is defined
+
+The initial contents of this registry are listed in the following table.
+
+| Value  | Name       | Reference                     |
+|:-------|:-----------|:------------------------------|
+| `0x00` | `reserved` | {{task-encoding}} of RFC XXXX |
+| `0x01` | `none`     | {{task-encoding}} of RFC XXXX |
+{: #dp-mechanism-id title="Initial contents of the Differential Privacy (DP) Mechanisms registry."}
+
+## DAP Sub-namespace for DAP
+
+> TODO Figure out how to ask IANA to register the errors in
+> {{urn-space-errors}}. See
+> https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap-taskprov/issues/34
+
+# Extending this Document
+
+The behavior of the `taskbind` extension may be extended by future documents
+that define:
+
+1. A new DAP batch mode
+1. A new VDAF
+1. A new DP mechanisms
+
+Such documents SHOULD include a section titled "Taskbind Considerations" that
+specifies an encoding of any configuration parameters associated with the DAP
+batch mode, VDAF, or DP mechanism. This encoding extends the `BatchConfig`
+structure, the `VdafConfig` structure, or the `DpConfig` structure
+respectively.
 
```

Comment:
Note that {{!DAP}} defines the registry for batch modes, {{!VDAF}} defines the registry for VDAFs, and this document defines the registry for DP mechanisms.